web-ts/README.md

```diff
@@ -36,15 +35,8 @@ _Make sure to have read and followed the steps in the [general README](../README
 
 - `Run > Edit Configurations > Defaults > Jest`:
   - Set `Configuration file` to `jest.config.js`
-- `Settings > Languages & Frameworks > JavaScript`:
-  - Choose `Flow` as Language version
-  - Set `Flow package or executable` to `<project_dir>/node_modules/flow-bin`
```

Comment:
```suggestion
- `Settings > Languages & Frameworks > Typescript`:
  - Check `TypeScript language service`
  - Set `TypeScript` package or executable to `<project_dir>/node_modules/typescript`
```

Reply:
Is this actually necessary? I thought intellij configures this automatically.

Reply:
I think we can remove this yes, the config is now actually included in the project config files.